cli/assets/templates/assemblyscript/asconfig.json

```diff
@@ -5,15 +5,18 @@
   "options": {
     "binaryFile": "build/cart.wasm",
     "runtime": "incremental",
+    "sourceMap": true,
```

Comment:
Should we put this only for debug builds?

Reply:
It automatically dropped for releases.

Reply:
source maps useful for all builds It's independent file. But if you have limit for bundle size better use it only debug builds